R/as_model_out_df.R

```diff
@@ -0,0 +1,211 @@
+#' Convert model output to a `as_model_out_tbl` class object.
```

Comment:
```suggestion
#' Convert model output to a `model_out_tbl` class object.
```

Reply:
Fixed in 45d3c0da56e26292c93ff8198fa310184f2239d5 👍